### PR TITLE
Release 3.3.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-release-3.3.1](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-release-3.3.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 3.3.1 (2020-03-11)
+
+- Fixed the country field component to not take up extra space below the component. [#59](https://github.com/blackbaud/skyux-lookup/pull/59)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: "...to eliminate extra space below the component."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+- Fixed the country field component to properly recognize if it is within a phone field component. [#59](https://github.com/blackbaud/skyux-lookup/pull/59)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: I'd remove "properly" and say "...to recognize when it is" instead. Is it _placed_ within the phone field component? If yes, then I'd say "...to recognize when it is placed within the phone field component.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-release-3.3.1](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-release-3.3.1)

**Pull request merged**